README.md

```diff
@@ -1,7 +1,7 @@
 # Calendar
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+- [DEMO LINK](https://Kyzenko.github.io/layout_calendar/)
```

Comment:
demo link doesn't work(
![image](https://user-images.githubusercontent.com/98764641/225267139-fc5c310d-1ef8-407e-b95a-722472805bb3.png)
try to redeploy and ask in the chat if it won't work